batch/parsers/testSuiteParserUtils.js

```diff
@@ -0,0 +1,71 @@
+var _ = require('lodash');
+
+export function normalizeTime(rawTime) {
+	let returnValue = 0;
+	if (rawTime) {
+		returnValue = Number.parseFloat(rawTime);
+	}
+	return returnValue.toFixed(2);
+}
+
+function parsedIntOrZero(stringToParse) {
+	return Number.parseInt(stringToParse) || 0;
+}
+export function testSuiteSummary(raw) {
+	return {
+		'tests': parsedIntOrZero(raw.tests),
+		'failures': parsedIntOrZero(raw.failures),
+		'skipped': parsedIntOrZero(raw.skipped),
+		'errors': parsedIntOrZero(raw.errors)
+	};
+}
+
+export function getFailureDataForSingleTest(raw) {
+	return {
+		'type': raw.$.type,
+		'message': raw.$.message,
+		'raw': raw._
+	};
+}
+
+export function tests(raw) {
+	return _(raw).map(function (test) {
+		var current = (test.failure || [])[0] || {$: {}, _: ''};
+
+		return _({
+			'name': test.$.name,
+			'time': normalizeTime(test.$.time),
+			'classname': test.$.classname,
+			'failure': getFailureDataForSingleTest(current)
+		}).tap(function (result) {
+			!test.failure && delete result.failure;
+		}).value();
+	}).value();
+}
+
+export function getOutputs(raw) {
+	const extras = {};
```

Comment:
I have a hard time understanding this variable name: the function is called `getOutputs`, yet we return `extras`. What about `result`, `outputs`, or make the varname longer and clearer as to why it's an “extra”?